src/service/user/cms_user_api.e

```diff
@@ -49,7 +49,8 @@ feature -- Status report
 				if is_admin_user (a_user) then
 					Result := True
 				else
-					Result := user_role_has_permission (authenticated_user_role, a_permission)
+					fixme ("Check how to handle this predefined role")
+						-- Result := user_role_has_permission (authenticated_user_role, a_permission)
```

Comment:
Please revert this change, there is really an "authenticated" user role ... in fact, as soon as a user is signed in, it has the authenticated user role.
This gives the possibility to set permission for all authenticated users.